src/main/java/ui/inputparser/InstructionType.java

```diff
@@ -0,0 +1,64 @@
+package ui.inputparser;
+
+/**
+ * The types of instructions that Kniaz can accept
+ * Each member of this enum has an alias, that represents the command typed into Kniaz
+ * E.g. the DEADLINE InstructionType is related to the command "deadline".
+ */
+public enum InstructionType {
+    TODO("todo", 1, 0, new String[]{}),
+    DEADLINE("deadline", 1, 1, new String[]{"by"}),
+    EVENT("event", 1, 2, new String[]{"from","to"}),
+    MARK("mark", 1, 0, new String[]{}),
+    UNMARK("unmark", 1, 0, new String[]{}),
+    LIST("list", 0, 0, new String[]{}),
+    QUIT("bye", 0 , 0, new String[]{}),
+    DELETE("delete", 1, 0, new String[]{}),
+    FIND("find",1,0,new String[]{}),
```

Comment:
Try to leave a space after a comma just like your above statements

Reply:
Was a new addition, didn't notice the variance in style, will fix!